content/tutorials/webrtc/basics/en/index.html

```diff
@@ -82,13 +87,12 @@ <h2 id="toc-where">Where are we now?</h2>
+<p><code><strong>getUserMedia</strong></code> is available in Chrome, Opera, Firefox and Edge. Take a look at the cross-browser demo at <a href="https://webrtc.github.io/samples/src/content/getusermedia/gum/" title="Simple cross-platform getUserMedia demo">demo</a> and Chris Wilson's <a href="http://webaudiodemos.appspot.com/" title="">amazing examples</a> using <code>getUserMedia</code> as input for Web Audio.</p>
 
-<p><strong><code>RTCDataChannel</code></strong> is supported by Chrome 25, Opera 18 and Firefox 22 and above.</p>
+<p><code><strong>RTCPeerConnection</strong></code> is in Chrome (on desktop and for Android), Opera (on desktop and in the latest Android Beta) and in Firefox. A word of explanation about the name: after several iterations, <code>RTCPeerConnection</code> is currently implemented by Chrome and Opera as <code>webkitRTCPeerConnection</code> and by Firefox as <code>mozRTCPeerConnection</code>. Other names and implementations have been deprecated. When the standards process has stabilized, the prefixes will be removed. There's an ultra-simple demo of Chromium's RTCPeerConnection implementation at <a href="https://webrtc.github.io/samples/src/content/peerconnection/pc1/" title="Simple cross-platform peerconnection demo">GitHub</a> and a great video chat application at <a href="https://apprtc.appspot.com" title="Video chat demo">apprtc.appspot.com</a>. This app uses <a href="https://github.com/webrtc/adapter" title="adapter.js JavaScript file">adapter.js</a>, a JavaScript shim, maintained Google with help from the <a href="https://github.com/webrtc/adapter/graphs/contributors" title="WebRTC/adapter contributors">GitHub community</a>, that abstracts away browser differences and spec changes.</p>
 
-<p>WebRTC functionality is available in Internet Explorer <a href="https://groups.google.com/forum/#!topic/discuss-webrtc/tKoh1wrI8ig" title="How to enable WebRTC functionality in Internet Explorer via Chrome Frame">via Chrome Frame</a>, and Skype (acquired by Microsoft in 2011) is reputedly <a href="http://gigaom.com/2012/06/26/skype-webrtc-web-client/" title="GigaOM blog post about Skype and WebRTC">planning to use WebRTC</a>. WebRTC has also been integrated with <a href="https://labs.ericsson.com/developer-community/blog/beyond-html5-conversational-voice-and-video-implemented-webkit-gtk" title="Ericsson article about WebKitGTK+">WebKitGTK+</a> and <a href="http://www.youtube.com/watch?v=Vm5ebKWKNE8" title="basysKom showcasing WebRTC based video chat in QML application">Qt</a> native apps.</p>
+<p><strong><code>RTCDataChannel</code></strong> is supported by Chrome M25, Opera 18 and Firefox 22 and above. Check out one of the data channel demos at <a href="https://webrtc.github.io/samples/" title="WebRTC samples">GitHub</a> to see it in action.
```

Comment:
I think you could just say 'supported by Chrome, Opera and Firefox'. The rest is ancient history :).